### ISSUE TITLE
Control whether user sees a popup

### ISSUE THREAD

**Issue by mdole** [artsy/clouds#54]
In recent Artsy for Galleries articles, we've been using a popup to gate the content and collect user information ([example](pages.artsy.net/online-art-terms-glossary.html)). Right now, the popup appears to everyone who goes to the page, regardless of whether they're a partner or whether we already have their information.

We'd like to change whether this popup appears based on certain lead criteria, i.e. not showing it to users whose subscription state indicates that they're a partner, and would ideally like to be able to easily change these criteria in the future for different scenarios. Cat [posted on the Marketo message boards](https://nation.marketo.com/message/170200-re-controlling-when-a-form-pops-up-on-marketo-landing-page?et=watches.email.thread#comment-170200) and the response indicates that this is possible and would benefit from some dev work. @craigspaeth is this something you'd be able to help out with?

[Current Marketo landing page with a form](https://app-ab14.marketo.com/#LP3427B2)

cc @cfinsness—feel free to add anything I missed.

**Comment by craigspaeth** [comment]
Yep looking at that thread (thanks for linking), this seems fairly straightforward. @mdole or @cfinsness Feel free to put an hour on the calendar to pair on this and I'd be happy to help.

**Comment by mdole** [comment]
thanks so much! added time to your cal tomorrow morning @craigspaeth; feel free to adjust if there's a different time you'd prefer
